docx/helper/helper.go

```diff
@@ -25,3 +27,43 @@ func GenerateXML(wordDoc *docx.Docx) (xmlDoc *xml.XmlDocument, err error) {
 	bytes := []byte(content)
 	return ParseXML(bytes)
 }
+
+// FillParagraph inserts the given content into the provided docx XML paragraph node. Note that newlines aren't respected - you'll need to create a new paragraph node for each.
```

Comment:
I think `FillParagraph` and  `AddMultiLineContent` deserve to be in `doc-template` but that  library needs a lot love and care before this would fit there